<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>679</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT</family>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="keyboardTracking">
          <bool>true</bool>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2016</year>
           <month>9</month>
           <day>13</day>
          </datetime>
         </property>
         <property name="maximumDateTime">
          <datetime>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
           <year>9999</year>
           <month>12</month>
           <day>31</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy.MM.dd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonArrival">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ПРИЛЁТ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonDeparture">
         <property name="font">
          <font>
           <family>Arial Rounded MT</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ВЫЛЕТ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="getFlightSchedule">
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ПОЛУЧИТЬ РАСПИСАНИЕ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="getLoadStatistic">
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ПОЛУЧИТЬ СТАТИСТИКУ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dataBaseStatus">
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
      </property>
      <property name="text">
       <string>ОТКЛЮЧЕНО</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
